tests/Dockerfile

```diff
@@ -1,35 +1,42 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
-RUN dnf install -y bc git go-toolset && \
-    git clone https://github.com/crobby/peak && \    
-    cd /peak && \
+ARG ORG=opendatahub-io
+ARG BRANCH=master
+
+RUN dnf install -y bc git go-toolset
+
+ENV HOME /root
+WORKDIR /root
+
+RUN git clone https://github.com/crobby/peak && \
+    cd peak && \
     git submodule update --init
 
-RUN mkdir -p /src && \
-    cd /src && \
-    git clone https://github.com/opendatahub-io/odh-manifests && \
-    chmod -R 777 /src
-
-ADD https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz /peak/ 
-RUN tar -C /usr/bin -xvf /peak/oc.tar.gz && \
-    chmod +x /usr/bin/oc
-
-COPY basictests /peak/operator-tests/odh-manifests/basictests
-COPY functionalitytests /peak/operator-tests/odh-manifests/functionalitytests
-COPY util /peak/operator-tests/odh-manifests
-COPY resources/kfctl_openshift.yaml /kfdef/
-COPY resources/operatorsetup /peak/
-COPY resources/ai_library_cr.yaml /peak/operator-tests/odh-manifests/resources/
-COPY scripts/install.sh /peak/
-COPY scripts/installandtest.sh /peak/
-
-RUN chmod -R 777 /kfdef && \
-    mkdir -p /.kube && \
```

Comment:
based on the test results from the PR to update the kube job, we might need to change permissions on /root (or something like that) to get the copying of the kubeconfig.
From the test logs:.....
```
mkdir: cannot create directory '/root/.kube': Permission denied
+ cp /var/run/secrets/ci.openshift.io/multi-stage/kubeconfig /root/.kube/config
cp: cannot create regular file '/root/.kube/config': No such file or directory
+ chmod 755 /root/.kube/config
chmod: cannot access '/root/.kube/config': No such file or directory
+ export KUBECONFIG=/root/.kube/config
+ KUBECONFIG=/root/.kube/config
+ oc new-project opendatahub
W0622 21:13:44.846228      29 loader.go:223] Config not found: /root/.kube/config
```